### DISCUSSION TITLE
Single product in multiple collections and variant images

### DISCUSSION THREAD

**Discussion by alinjie** [medusajs/medusa#980] [Q&A] [answered]
Hi!

Recently found Medusa through Reddit and it seems like a really cool project!

Two things that are essential in order for me to go with Medusa as my go-to tool for headless e-commerce are:
- The ability to add a product into multiple collections
- The ability to add one or multiple images to a variant

Are there any chances that these features will be added in the future?

**Comment by srindom** [comment, marked as answer]
Hi Ali! Thanks for the kind words :)

The short answer is yes to both - the core team is currently working on a redesign of the admin system and we expect to start working on enhancing the core with new features and tweaks like the ones you suggest here once the new admin system is released!

**Reply by alinjie** [reply]
Super cool - looking forward to it! 🎉

**Comment by aihdeveloper** [comment]
Hi Is there an update for these two features? Really looking forward to it!

**Reply by olivermrbl** [reply]
We are currently working on Product Categories, that introduce a many-to-many relationship with products. You can follow the technical design process [here](https://github.com/medusajs/medusa/discussions/2873).
